be/src/runtime/mem_pool.cpp

```diff
@@ -115,20 +131,10 @@ bool MemPool::find_chunk(size_t min_size, bool check_limits) {
     }
 
     chunk_size = BitUtil::RoundUpToPowerOfTwo(chunk_size);
-    if (check_limits) {
```

Comment:
check_limit = false, which means that the memory limit is not checked when calling MemTracker::consume.

In fact, most MemPools in Doris currently use check_limit = false when applying for memory. Hereafter, when refactoring MemPool, it can be sorted out uniformly.